### DISCUSSION TITLE
Edit jetstream replicas count

### DISCUSSION THREAD

**Discussion by astelmashenko** [nats-io/nats-server#4263] [Q&A] [answered]
Is it possible to change jetstream replicas from 1 to 3?
I'm trying to execute:
`nats s edit --replicas=3 my-stream`
getting the next output:
```
Differences (-old +new):
  api.StreamConfig{
        ... // 10 identical fields
        Storage:  s"File",
        Discard:  s"Old",
-       Replicas: 1,
+       Replicas: 3,
        NoAck:    false,
        Template: "",
        ... // 8 identical fields
  }
? Really edit Stream my-stream Yes
nats: error: could not edit Stream my-stream: Replicas configuration can not be updated (10061)
```

There is a cluster of nats servers with three instances and there are other streams with replicas=3, but those were created with replicas=3 from start. And I'm trying to change replicas to those which were created with replicas=1 from start to be replicas=3.

**Comment by Jarema** [comment]
You definately can update replica count in a stream.

What version of the server and cli are you running?

**Comment by astelmashenko** [comment]
Nats cli is 0.0.28 and server version is 2.7.2

**Comment by Jarema** [comment, marked as answer]
That's a very old server version.
I would definitely upgrade to the latest one (2.9.19) and try again.

**Comment by astelmashenko** [comment]
Jarema, thank you for quick answer we will try to upgrade and come back later with a result.

**Comment by astelmashenko** [comment]
It worked, thank you.
